.pre-commit-config.yaml

```
@@ -38,13 +35,13 @@ repos:
   - repo: https://github.com/akaihola/darker
     rev: v2.1.1
     hooks:
-    -   id: darker
-        args: ["--check"]
+      - id: darker
+        args: ["--check", "--diff", "--color"]
```

We don't want "--diff" for the pre-commit, because `--diff` only shows the recommended update but does not update the file, and we want the pre-commit to update the file.

```suggestion
        args: ["--check", "--color"]
```

---

According to https://github.com/akaihola/darker the `--check` already does not update the file, right?
And, in that case, if writing to the files directly was intended. Both `--check` and. `--diff` needs to be removed, I guess.

---

Seems so indeed. I assumed since `--check` was there already, that it wrote to files. Perhaps as a pre-commit hook it ignores the `--check` arg. In any case, if both need to be removed to be correct, then we should do that.

---

Now Looking at the full darker hooks:
```yaml
  - repo: https://github.com/akaihola/darker
    rev: v2.1.1
    hooks:
      - id: darker
        args: ["--check", "--diff", "--color"]
        stages: [push]
      - id: darker
        stages: [commit]
```

 In the current configuration:

- When `commit`, darker will automatically format the Python files
- When `push`, darker will only check if files need formatting (using --check) and show the diffs that would be applied (--diff), but won't actually modify them. And this case of finding issues at push but not at commit, I think happens on cases like merging & rebasing... So, I think, it make sense to give a notification and prevent pushing in those cases so the pusher would have a look and modify manually. 

This was the old behaviour and still work as it was. The ` "--diff", "--color"` just makes the output nicer and more informative.

And so, all seems good to me. Let me know if you have some suggestions please.